### ISSUE TITLE
Additional spaces are added before comments and after the semicolons of RAML keywords

### ISSUE THREAD

**Issue by deiteris** [aml-org/als#522]
Your issue may already be reported! Please search on Github issues before creating one.

* **I'm submitting a ...**
    - [x] bug report
    - [ ] feature request
    - [ ] question
    - [ ] other

* **What is the current behavior?**

When using the formatting feature, comments are moving by set amount of spaces and additional space is adding after the semicolon of RAML keywords.

* **If the current behavior is a bug, please provide the minimal steps to reproduce on ALS**

1. Create a simple document containing a comment and multiline description:
    ```yaml
    #%RAML 1.0
    
    title: Sample API # Test comment
    
    types:
      TestType:
        description: |
          Multiline
          description
    ```
2. Execute the `Format Document` command.
3. Notice that additional spaces were added:
    ```yaml
    #%RAML 1.0
    
    title: Sample API   # Test comment
    
    types:
      TestType:
        description:  |-
          Multiline
          description
    ```

* **What is the expected behavior?**

Additional spaces should not be added before comments and after semicolons of keywords.

* **Please tell us about your environment:**

    - ALS Version: 3.2.6
    - ALS Distribution: JVM
    - OS: Windows 10 20H2
    - IDE: Visual Studio Code 1.55.2

* **Other information** (e.g. detailed explanation, stack traces, related issues, workarounds, links for us to have context, eg. StackOverflow, Gitter, etc)

**Comment by llibarona** [comment]
I created an internal ticket so we can prioritise this fix

link to internal ticket: https://www.mulesoft.org/jira/browse/ALS-1486

**Comment by agusayerza** [comment]
#608 contains the fix implemented in #606

**Comment by agusayerza** [comment]
The fix is in develop and should be released with version v4.1.1
